elements/lisk-p2p/src/utils.ts

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { hash } from '@liskhq/lisk-cryptography';
+import { isIPv4 } from 'net';
+
+const SECRET_BUFFER_LENGTH = 4;
+const NETWORK_BUFFER_LENGTH = 1;
+const PREFIX_BUFFER_LENGTH = 1;
+const BYTES_4 = 4;
+const BYTES_16 = 16;
+const BYTES_64 = 64;
+const BYTES_128 = 128;
+const NEW_PEERS = 'new';
+const TRIED_PEERS = 'tried';
+
+export enum NETWORK {
+	NET_IPV4 = 0,
+	NET_PRIVATE,
+	NET_LOCAL,
+	NET_OTHER,
+}
+
+/* tslint:disable no-magic-numbers */
+export const getGroup = (
+	address: string,
+	groupNumber: number,
+): number | undefined => {
+	if (groupNumber > 3) {
+		return undefined;
+	}
+
+	const group = parseInt(address.split('.')[groupNumber], 10);
+
+	if (!Number.isInteger(group)) {
```

Comment:
That's what it was originally and it's a good point, but after @ishantiw comment I thought some of these `util` functions can be used by the user independently for their own purposes, in which case it might help to re-validate. What do you think?